packages/roosterjs-content-model-plugins/lib/edit/EditPlugin.ts

```diff
@@ -54,14 +62,34 @@ export class EditPlugin implements EditorPlugin {
     private disposer: (() => void) | null = null;
     private shouldHandleNextInputEvent = false;
     private selectionAfterDelete: DOMSelection | null = null;
-    private handleNormalEnter = false;
+    private handleNormalEnter: (editor: IEditor) => boolean = (editor: IEditor) => false;
 
     /**
      * @param options An optional parameter that takes in an object of type EditOptions, which includes the following properties:
      * handleTabKey: A boolean that enables or disables Tab key handling. Defaults to true.
      */
     constructor(private options: EditOptions = DefaultOptions) {}
 
+    private createNormalEnterChecker(result: boolean) {
+        return result ? () => true : () => false;
+    }
+
+    private getHandleNormalEnter(editor: IEditor) {
+        switch (typeof this.options.shouldHandleEnterKey) {
+            case 'function':
+                return this.options.shouldHandleEnterKey;
+                break;
+            case 'boolean':
+                return this.createNormalEnterChecker(this.options.shouldHandleEnterKey);
+                break;
+            default:
+                return this.createNormalEnterChecker(
```

Comment:
I did a bit different than what you proposed, I added back the experimental feature check, so it do not break the code of the user that has the experimental features enabled.